### PR TITLE
Form: update label paddings in Fluent

### PR DESCRIPTION
**Pull request by iBat**

### CODE REVIEW

File: packages/devextreme/scss/widgets/fluent/form/_index.scss [hunk 1]
```diff
@@ -41,14 +41,18 @@
 .dx-layout-manager {
   .dx-field-item {
     &:not(.dx-last-col) {
-      padding-right: 20px;
+      padding-right: 8px;
```
Comment (marker-dao) [inline packages/devextreme/scss/widgets/fluent/form/_index.scss]: Let's use scss variables.

Comment (EugeniyKiyashko) [reply packages/devextreme/scss/widgets/fluent/form/_index.scss]: Extract it please in variable as follows:
`
$form-layout-manager-padding: 8px;
`

File: packages/devextreme/scss/widgets/fluent/form/_sizes.scss [hunk 2]
```diff
@@ -9,24 +9,30 @@ $fluent-form-group-caption-font-size: null !default;
...
-  $fluent-form-default-top-padding: 26px !default;
+  $fluent-form-default-top-padding: 24px !default;
+
+  $fluent-form-label-font-size: 14px !default;
+  $fluent-form-label-icon-size: 14px !default;
 }
 
 @else if $size == "compact" {
   $fluent-form-group-caption-bottom-margin: 4px !default;
   $fluent-form-group-caption-bottom-padding: 10px !default;
```
Comment (EugeniyKiyashko) [inline packages/devextreme/scss/widgets/fluent/form/_sizes.scss]: I suggest that paddings should be 8px here. Check please other values of variables for compact and default sizes

**Comment by EugeniyKiyashko** [comment]
Please add changes from https://github.com/DevExpress/DevExtreme/pull/25650 in your PR. We will get all screenshots for Form component in Fluent. A new screenshots should be approved by designer. If will require additional changes in styles, let's do it.

**Review by marker-dao — DISMISSED** [review]

**Review by EugeniyKiyashko — APPROVED** [review]

**Review by Lykoi18 — APPROVED** [review]

**Pull request merged**